DMX.Portal.Web/Views/Bases/ButtonBase.razor.cs

```diff
@@ -0,0 +1,18 @@
+﻿// --------------------------------------------------------------- 
+// Copyright (c) Microsoft Corporation. All rights reserved. 
+// ---------------------------------------------------------------
+
+using Microsoft.AspNetCore.Components;
+
+namespace DMX.Portal.Web.Views.Bases
+{
+    public partial class ButtonBase
+    {
+        [Parameter]
+        public string Text { get; set; }
+
+        [Parameter]
+        public string ClassName { get; set; }
+

```

Comment:
@geetika-jain we need to also implement the button click event.

here's an example:
https://github.com/hassanhabib/OtripleS.Portal/blob/master/OtripleS.Portal.Web/Views/Bases/ButtonBase.razor.cs